### PR TITLE
Fix inviting multiple players in one message

### PR DESCRIPTION
**Pull request by CuzImClicks**
Breefing, Ealman and DeathStreeks complained about it in their streams a lot
(When inviting multiple players in one command hypixel creates an invalid invite)

### CODE REVIEW

File: src/main/kotlin/gg/skytils/skytilsmod/commands/impl/PartyCommand.kt [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+        if (!args.first().endsWith(",")) {
+            UChat.say("/p ${args.joinToString(" ")}")
+            return
+        } else {
+            Thread {
+                args.forEach {
+                    UChat.say("/p ${it.replace(",", "").replace(" ", "")}")
+                    Thread.sleep(400)
+                }
+            }.start()
```
Comment (TakoTheDev) [inline src/main/kotlin/gg/skytils/skytilsmod/commands/impl/PartyCommand.kt]: Skytils.sendMessageQueue.add("/command to run") exists

Comment (TakoTheDev) [reply src/main/kotlin/gg/skytils/skytilsmod/commands/impl/PartyCommand.kt]: Also why replace " "?

**Review by TakoTheDev — COMMENTED** [review]
Just make a list of sub-commands (listOf("i", "invite"))

File: src/main/kotlin/gg/skytils/skytilsmod/commands/impl/PartyCommand.kt (hunk 1 above)
Comment (CuzImClicks) [reply src/main/kotlin/gg/skytils/skytilsmod/commands/impl/PartyCommand.kt]: > Skytils.sendMessageQueue.add("/command to run") exists

uses a delay of 250ms while hypixel does not allow inviting players that quickly

> Also why replace " "?

safety

**Comment by My-Name-Is-Jeff** [comment]
closed on accident but i don't like how it replaces any commands

**Pull request closed without merging**